src/Compilers/CSharp/Portable/Symbols/Source/SourceSimpleParameterSymbol.cs

```diff
@@ -27,6 +27,19 @@ internal override ConstantValue ExplicitDefaultConstantValue
             get { return null; }
         }
 
+        internal override bool IsNullChecked
+        {
+            get
+            {
+                var node = this.GetNonNullSyntaxNode();
+                if (node is ParameterSyntax param)
+                {
+                    return param.ExclamationToken.IsMissing;
```

Comment:
This returns true if the exclamation token is missing. I think you want the inverse.